console/src/components/Services/Data/Common/Components/TableRow.tsx

```diff
@@ -7,7 +7,11 @@ import {
 import styles from '../../../../Common/TableCommon/Table.scss';
 import { TypedInput } from './TypedInput';
 
-const getColumnInfo = (col: TableColumn, prevValue?: unknown) => {
+const getColumnInfo = (
+  col: TableColumn,
+  prevValue?: unknown,
+  clone?: Record<string, any>
```

Comment:
can we have the following type here? and use this same type for `TableRowProps`
```suggestion
  clone?: Record<string, number | string | boolean>
  // or 
  clone?: Record<string, unknown>
```
we will try to avoid `any` as much as possible

Reply:
@soorajshankar, we don't know what is there. There can be literally `any` type. It won't cause less type safety, we just allow it to be anything. We can have json/jsonb, array data types, etc, right? We don't much care what exactly this thing is. This is a value stored in a table row.

Reply:
`unknown` will work here.

Reply:
changing my suggested code to only `unknown` suggestion

Reply:
However, note that `unknown` is not always better than any. When we don't care about the type (type can be anything) and we annotate it with `unknown` then it may be very hard to work with it. You basically can't do anything with an unknown, so every time you want to perform something on the value annotated as unkown you'd need add types assertion `as any` anyway.